pages/web-sites.md

```diff
@@ -120,8 +120,14 @@ For other sites:
 
 To create a tag, use `npm version [major | minor | patch]`.
 
-Afterwards, make sure to push both version change commit and the tag:
-`git push --tags origin master`
+Afterwards, make sure to push both version change commit and the tag to the main jQuery repo.
+
+<div class="warning">You'll want to check your remote setup using <code>git remote -v</code> and check which remote is used for the main jQuery repo.</div>
```

Comment:
Perhaps:

The example below uses `upstream` for the jQuery organization on GitHub. Use <code>git remote -v</code> to ensure you're pushing to the correct remote repo.